### DISCUSSION TITLE
Problems of verifying my Teacher account.

### DISCUSSION THREAD

**Discussion by tidegu** [community/community#133704] [GitHub Education] [answered]
### Hi everyone,

I am a lecturer from a university in China, and trying to verify my GH education as a teacher. The application page asks me to provide two proofs, one of which is the ID or other proof that shows the terms of my employement, the other asks me to answer why I'm out of campus.
For the first, my university ID does not contain the validity date, nor any documents shows that. It because in China, the contract is long term by default, which means unless you resign or be dismissed from the university, you'll be permantely a staff. The only document that I have is the contract, which firstly do not have the letterhead (of course it has stamps on) and secondly it does not state the end date, rather a starting date and 10 years term as kinda probation (not really a probation but it means you cannot leave the university without paying back the initial support fee to the university, unless the university dismisses you), this term also does not mean you have only 10 years contract to the university as the employement will automatically roll without any process needed once  the 10 year ends - HR won't let you sign off any further document and you don't have to / can't sign any further document manually. Even worse, it's all in Chinese and I don't know if it can be used as proof.
The other question is to answer why I'm not  on campus, the reason is very simple - You may know China network blocks many foreign websites, (includes Google, Twitter, Facebook etc), for GH, the network is very unstable, sometimes it allows me to browse but many times I can't browse. That's why we have to use VPN to gain stable access (not only for GH but also for Google Scholar and many other websites). I don't know how to proof that I'm using VPN because I set up the proxy on my gateway, it's transparent from my computer. I'm OK to provide a screenshot of the setup page on my proxy but the plugin is all in Chinese anyway - not sure if it can be accepted. Also, I tried to use my university network when Github is accessible (without VPN),  it also questioning why I'm out of campus.
I don't know if I can submit whatever I have to  the application page, as it says once the application is made, I cannot modify - I'm afraid if it's one-off application and if it's failed I can't reapply.
BTW, I have verified my university email address and it's on my GH profile, now the homepage shows the notification that encourage me join GH education.
Many thanks in advance!

**Comment by wsong83** [comment]
Same problem here. Put it bluntly, the application pages and the exact restrictions are simply unfriendly towards anyone trying to apply in China.

**Comment by SohanKafle** [comment, marked as answer]
Hi [tidegu](https://github.com/tidegu),

I understand the challenges you are facing with providing the required proof for verifying your GitHub Education account as a teacher. Here's a concise plan to help you navigate the verification process:

<h2>Proof of Employment:</h2>
<b>1. University ID:</b> If it lacks validity dates, you can supplement it with other documents.<br>
<b>2. Contract:</b> Translate the key sections into English, including the starting date and the terms explaining the 10-year period and automatic roll-over. Highlight the official stamp as evidence of authenticity.

<h2>Explanation for Using VPN:</h2>
</b>1. Network Issues:</b> Write a clear explanation in English stating that the instability of internet access in China necessitates the use of a VPN to access GitHub and other essential academic resources.<br>
<b>2. Proxy Setup:</b> Provide screenshots of your VPN or proxy setup, even if in Chinese, along with a brief English description of each step.

<h2>Combining Documents:</h2>
1. Combine your translated contract sections and explanation into a single PDF.<br>
2. Include screenshots of your university network setup, proxy setup, and a written explanation.

<h2>Submission:</h2>
1. Submit all the compiled documents through the GitHub Education verification page.<br>
2. Clearly explain your situation in the "Why are you off-campus?" section.

<h2>Verification Email:</h2>
Make sure your university email address is verified and listed on your GitHub profile. This adds an extra layer of authenticity.

<h2>Support Contact:</h2>
If the initial application is unsuccessful, contact GitHub Education support directly through the support page or email ([education@github.com](mailto:education@github.com)) explaining your situation and providing the documents.

**Reply by samus-aran** [reply]
Would like to correct the above reply: **Support do not provide manual reviews of applications** 

Contact Support **only** if you experience a bug, school information is incorrect and requires updating or any other queries regarding partnerships or Classroom.

**Comment by fbuendia-upv** [comment]
Hi,
my name is Felix and I'm an associate professor at a Spanish university.
I was trying several times to provide the required information to access the GitHub Education as a teacher but the system rejected every image (jpg) I was sending to verify my academic status.
I guess that it is a problem about the automatic recognition for the upload images and I wondered if there is some alternative way to send this academic status information.
Best regards
